web/source/dom/domManager.ts

```diff
@@ -1241,8 +1241,14 @@ namespace com.keyman.dom {
     set lastActiveElement(Pelem: HTMLElement) {
       DOMEventHandlers.states._lastActiveElement = Pelem;
 
-      if(this.lastActiveElement == null && this.activeElement == null) {
-        this.keyman.osk.hideNow(); // originally from a different one, seemed to serve the same role?
+      const osk = this.keyman.osk;
+      if(osk) {
+        if(this.lastActiveElement == null && this.activeElement == null) {
+          if(this.keyman.osk.activeTarget) {
+            this.keyman.osk.activeTarget = null;
+          }
```

Comment:
```suggestion
          if(osk.activeTarget) {
            osk.activeTarget = null;
          }
```

Also, do we actually need the test here? Couldn't we just always set it to `null`?